### PR TITLE
Ta64 2

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Some small fixes needed for 64-bit TAs

### CODE REVIEW

File: lib/libutee/arch/arm/sub.mk [hunk 1]
```diff
@@ -2,4 +2,5 @@ cppflags-y += -I$(sub-dir)/../..
 
 srcs-y += user_ta_entry.c
 srcs-y += utee_misc.c
-srcs-y += utee_syscalls_asm.S
+srcs-$(CFG_ARM32_user_ta) += utee_syscalls_a32.S
+srcs-$(CFG_ARM64_user_ta) += utee_syscalls_a64.S
```
Comment (pascal-brand-st-dev) [inline lib/libutee/arch/arm/sub.mk]: Could it be stated more explicitly in the commit message, or later in documentation, that libutee is compiled for a given 32bits or 64bits mode, but right now, we do not provide both versions of the library?

**Comment by pascal-brand-st-dev** [comment]
Apart a minor comment,

    Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

File: lib/libutee/arch/arm/sub.mk (hunk 1 above)
Comment (jenswi-linaro) [inline lib/libutee/arch/arm/sub.mk]: So far only 32-bit compilation is supported, here we're just providing some prerequisites (which I have verified with other patches that they are OK).

I'll update the commit message.

**Comment by jenswi-linaro** [comment]
Update with tags.

**Pull request merged**